dot/rpc/subscription/websocket.go

```diff
@@ -209,6 +212,12 @@ func (c *WSConn) initStorageChangeListener(reqID float64, params interface{}) (u
 	return myObs.id, nil
 }
 
+func (c *WSConn) unsubscribeStorageListener(params interface{}) {
+	id := params.([]interface{})[0].(float64)
+	observer := c.Subscriptions[uint(id)].(state.Observer)
```

Comment:
This may panic because there could be different implementation of `Listener` interface.
```
observer, ok := c.Subscriptions[uint(id)].(state.Observer)
if !ok {
   return
}
```
Also, we should only unsubscribe storage observers.